ui/apps/platform/src/Containers/SystemHealth/Components/IntegrationsHealth.tsx

```diff
@@ -1,89 +1,48 @@
-    });
-
-    // The item border matches the widget header border.
-    if (integrationsFiltered.length !== 0) {
-        const { Icon, fgColor } = styleUnhealthy;
-        return (
-            <ul className="leading-normal pt-1 w-full">
-                {integrationsFiltered.map(({ id, name, label, lastTimestamp }) => (
-                    <li className="border-b border-base-300 px-2 py-1" key={id}>
-                        <div className="flex w-full">
-                            <div className={`flex-shrink-0 ${fgColor}`}>
-                                <Icon className="h-4 w-4" />
-                            </div>
-                            <div className="ml-2 flex-grow">
-                                <div className="font-700" data-testid="integration-name">
-                                    {name}
-                                </div>
-                                {label && label !== name && (
-                                    <div className="text-base-500" data-testid="integration-label">
-                                        {label}
-                                    </div>
-                                )}
-                                {lastTimestamp && (
-                                    <div>
-                                        <span>Last contact:</span>{' '}
-                                        <span data-testid="last-contact">
-                                            {getDateTime(lastTimestamp)}
-                                        </span>
-                                    </div>
-                                )}
-                            </div>
-                        </div>
-                    </li>
-                ))}
-            </ul>
-        );
-    }
-
-    const { Icon, fgColor } = styleHealthy;
-    const nIntegrations = integrationsMerged.length;
-    let text = 'No configured integrations';
-    if (nIntegrations !== 0) {
-        if (nHealthy === nIntegrations) {
-            text = `${nHealthy} healthy ${pluralize('integration', nHealthy)}`;
-        } else {
-            text = `${nHealthy} / ${nIntegrations} healthy integrations`; // cannot both be singular
-        }
+const IntegrationsHealth = ({ integrations }: Props): ReactElement => {
+    if (integrations.length === 0) {
+        return <></>;
     }
-
     return (
-        <div className={`flex flex-col h-full justify-center w-full ${fgColor}`}>
-            <div className="flex justify-center mb-2">
-                <Icon className="h-6 w-6" />
-            </div>
-            <div className="leading-normal px-2 text-center" data-testid="healthy-text">
-                {text}
-            </div>
-        </div>
+        <TableComposable variant="compact">
+            <Thead>
+                <Tr>
+                    <Th width={20}>Name</Th>
+                    <Th width={20}>Label</Th>
+                    <Th width={45}>Error message</Th>
+                    <Th width={15}>Date</Th>
```

Comment:
This reminds me to look at table column widths in other cards on System Health page.